Realm/RLMObjectStore.h

```diff
@@ -34,6 +34,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 void RLMVerifyHasPrimaryKey(Class cls);
 
+void RLMVerifyInWriteTransaction(RLMRealm *const realm);
```

Comment:
This doesn't appear to be needed in the header?

Reply:
this exposes the method that allows you see if a Realm is in a transaction, the other `RLMVerifyInWriteTransaction` verifies if an attached object is in a transaction